generate/src/contract/events.rs

```diff
@@ -70,6 +235,8 @@ fn expand_filter(event: &Event) -> Result<TokenStream> {
     })
 }
 
+/// Expands into the `all_events` method on the root contract type if it
+/// contains events. Expands to nothing otherwise.
```

Comment:
So all_events will continue to return events untyped? Not int his PR, but does it make sense to autogenerate an overarching all events enum that can have each event as a variant?

Reply:
Yes, we will generate an enum with all even variants (including anonymous events) do it that way. Currently the runtime (the `ethcontract::contract::*` stuff) doesn't quite support it. Its definitely coming next.